### ISSUE TITLE
Odd error message - code 400? 

### ISSUE THREAD

**Issue by dbrb2** [asantaga/wiserHomeAssistantPlatform#407]
I've noticed this in my logs recently. However, the integration seems to be working fine...so perhaps it doesn't matter! 

```
Response error trying to communicate with Wiser Hub 192.168.1.249 
for url http://192.168.1.249:80/data/v2/domain/. 

Error is 400, message="Data after `Connection: close`:\n\n b'0'\n ^", 
url=URL('http://192.168.1.249:80/data/v2/domain/')
```

**Comment by msp1974** [comment]
This seems to be an issue with HA 2023.12 updating the version of aiohttp.  This happened in a 2023.11 release and was rolled back.

The hub seems to frequently/occasionally send extra data at the end of the response and the aiohttp library (updated one) raises this as a 400 error and request to hub fails.

This is why you see this behaviour.

I am looking how to resolve.

**Comment by plord12** [comment]
Same here -

```
2023-12-07 10:51:03.876 WARNING (MainThread) [custom_components.wiser.coordinator] Response error trying to communicate with Wiser Hub wiserhub.lan for url http://wiserhub.lan:80/data/v2/status/. Error is 400, message="Data after `Connection: close`:\n\n b'0'\n ^", url=URL('http://wiserhub.lan:80/data/v2/status/')
```

Running curl from my desktop gives instant response.

**Comment by silfax** [comment]
Yes, I can confirm it now works for me. Thanks a lot, that was really fast!

**Comment by msp1974** [comment]
Great news closing this issue as resolved in v3.4.0
